### PR TITLE
Add dev environment installation instructions for Arch Linux-based systems

### PR DESCRIPTION
**Pull request by inpv**
### Motivation and context

Since Arch Linux-based systems are quite popular, especially in home use and on older hardware, I assume that cvat installation instructions for this type of medium could benefit the community.

### How has this been tested?

Tested the installation on a desktop home system (CPU i5 Skylake 6600K, 32 GB RAM, GPU AMD RX 480, pure Arch with XFCE), which is updated daily. The repos enabled are core, extra, community and multilib. AUR is managed by [pikaur](https://github.com/actionless/pikaur).
The packages and other related steps required for installation were found by trial and error.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/cvat-ai/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/cvat-ai/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/contributing/development-environment.md [hunk 1]
```diff
@@ -27,6 +27,28 @@ description: 'Installing a development environment for different operating syste
...
+  pikaur -Syu
+  ```
+
+  ```bash
+  # Install the required dependencies:
+  sudo pacman -S base-devel curl git redis cmake gcc python python-pip tk libldap libsasl pkgconf ffmpeg geos openldap python-lda
+  ```
+
+  ```bash
+  # Since migration only works with Python<=3.9, because the Iterable class has been removed from collections in Python 3.10, install Python<=3.9 if you don’t have it:
```
Comment (zhiltsov-max) [inline site/content/en/docs/contributing/development-environment.md:43]: Please update the wording in the comment from the point that Python 3.9 is expected and the only supported by the CVAT server.

Comment (inpv) [reply site/content/en/docs/contributing/development-environment.md]: Sure, specified the only supported Python version in the comment.

**Comment by inpv** [comment]
Specified the only supported Python version in the new commit.

**Review by zhiltsov-max — APPROVED** [review]
LGTM, thanks for the contribution!

File: site/content/en/docs/contributing/development-environment.md [hunk 2]
```diff
@@ -27,6 +27,28 @@ description: 'Installing a development environment for different operating syste
...
+  pikaur -Syu
+  ```
+
+  ```bash
+  # Install the required dependencies:
+  sudo pacman -S base-devel curl git redis cmake gcc python python-pip tk libldap libsasl pkgconf ffmpeg geos openldap python-lda
+  ```
+
+  ```bash
+  # Since migration only works with Python 3.9, because the Iterable class has been removed from collections in Python 3.10, install Python 3.9 if you don’t have it:
```
Comment (zhiltsov-max) [inline site/content/en/docs/contributing/development-environment.md:43]:
```suggestion
  # CVAT supports only Python 3.9, so install it if you don’t have it:
```

**Pull request merged**